src/requirements/__test__/__snapshots__/requirement-data-id.test.ts.snap

```diff
@@ -9,7 +9,6 @@ Array [
   "College-AG-Physical and Life Sciences",
   "College-AG-Quantitative Literacy",
   "College-AG-Social Sciences and Humanities",
-  "College-AG-Total Academic Credits",
```

Comment:
snapshot is updated as expected, since there are no longer part of the requirement json any more. Since it's before launch, no migration is necessary. For more background, see https://github.com/cornell-dti/course-plan/pull/222